.github/BOTMETA.yml

```
@@ -1168,7 +1168,7 @@ files:
   $modules/web_infrastructure/jenkins_script.py:
     maintainers: hogarthj
   $modules/web_infrastructure/jira.py:
-    maintainers: Slezhuk tarka pertoft DWSR
+    maintainers: Slezhuk tarka pertoft
```

Since you're an author of the module, you have to add yourself to the ignore list:
```suggestion
    maintainers: Slezhuk tarka pertoft
    ignore: DWSR
```